projects/avodata/firstStage.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import math
+
+def factorial(n):
+	return int(math.gamma(n + 1))
+
+
+def nChoosek(n, k):
+	return factorial(n) / (factorial(k) * factorial(n - k))
+
+
+def find_c_and_a(m):
```

Comment:
Please document the shape of m, and the shape of the outputs.

Reply:
done. I think.. this is what you mean?

Reply:
Yes, great. It will change a bit when we refactor.